src/thunderbots/backend_output/backend.cpp

```diff
@@ -0,0 +1,5 @@
+#include "backend_output/backend.h"
+
+void Backend::sendPrimitives(const std::vector <Primitive> &primitives) {
```

Comment:
Do we maybe want to add "using namespace std;" to the top of our files to reduce repetition of "std::"?

Reply:
Generally `using namespace std;` is considered bad practice, especially if we're using it in conjunction with a large framework like ROS that implements a ton of its own datatypes.

(see https://stackoverflow.com/questions/1452721/why-is-using-namespace-std-considered-bad-practice)

Reply:
Sounds good to me